tests/foreman/cli/test_contentcredentials.py

```diff
@@ -669,223 +684,38 @@ def test_positive_update_key_for_repo_from_product_with_repos(module_org):
     # Create repositories and assign them to the product
     repos = [make_repository({'product-id': product['id']}) for _ in range(gen_integer(2, 5))]
     # Associate gpg key with a single repository
-    Repository.update(
+    module_target_sat.cli.Repository.update(
         {'gpg-key-id': gpg_key['id'], 'id': repos[0]['id'], 'organization-id': module_org.id}
     )
     # Verify gpg key was associated
-    repos[0] = Repository.info({'id': repos[0]['id']})
+    repos[0] = module_target_sat.cli.Repository.info({'id': repos[0]['id']})
     assert repos[0]['gpg-key']['name'] == gpg_key['name']
     # Update the gpg key
     new_name = gen_choice(list(valid_data_list().values()))
-    ContentCredential.update(
+    module_target_sat.cli.ContentCredential.update(
         {'name': gpg_key['name'], 'new-name': new_name, 'organization-id': module_org.id}
     )
     # Verify changes are reflected in the gpg key
-    gpg_key = ContentCredential.info({'id': gpg_key['id'], 'organization-id': module_org.id})
+    gpg_key = module_target_sat.cli.ContentCredential.info(
+        {'id': gpg_key['id'], 'organization-id': module_org.id}
+    )
     assert gpg_key['name'] == new_name
     # Verify changes are reflected in the associated repository
-    repos[0] = Repository.info({'id': repos[0]['id']})
+    repos[0] = module_target_sat.cli.Repository.info({'id': repos[0]['id']})
     assert repos[0]['gpg-key'].get('name') == new_name
     # Verify changes are not reflected in the product
-    product = Product.info({'id': product['id'], 'organization-id': module_org.id})
+    product = module_target_sat.cli.Product.info(
+        {'id': product['id'], 'organization-id': module_org.id}
+    )
     assert product['gpg']['gpg-key'] != new_name
     # Verify changes are not reflected in the rest of repositories
     for repo in repos[1:]:
-        repo = Repository.info({'id': repo['id']})
+        repo = module_target_sat.cli.Repository.info({'id': repo['id']})
         assert repo['gpg-key'].get('name') != new_name
 
 
-@pytest.mark.tier2
-def test_positive_delete_key_for_empty_product(module_org):
```

Comment:
why removing this tests, is this intentional? saw your comment "this is blocked since 6.13.0 and they all fail for it:", can you explain?

Reply:
@Gauravtalreja1 There is a test for the BZ that David referenced `test_positive_block_delete_key_in_use`